workbench-core/accounts/src/services/projectService.ts

```diff
@@ -56,20 +62,143 @@ export default class ProjectService {
     }
   }
 
+  // TODO--delete after dynamic Authz
+  private _mockGetUserGroups(): string[] {
```

Comment:
I believe I would still have to mock something for that to work in whatever I pass into the constructor. That was not in scope for this ticket so I am going to go with what works. This will be taken out shortly anyway.